### PR TITLE
Release 20200902

### PR DESCRIPTION
**Pull request by ElenaFdR**
## Review checklist template

- [x] Correct Markdown and formatting
- [x] All new files are linked in indices
- [x] Front matter is present and correct
- [x] Bullets and numbered lists are correct
- [x] All content reviewed

-----
[View rendered CONTRIBUTING.md](https://github.com/publiccodenet/about/blob/release-20200902/CONTRIBUTING.md)
[View rendered activities/communication/index.md](https://github.com/publiccodenet/about/blob/release-20200902/activities/communication/index.md)
[View rendered activities/events/attending-events.md](https://github.com/publiccodenet/about/blob/release-20200902/activities/events/attending-events.md)
[View rendered activities/workshops/index.md](https://github.com/publiccodenet/about/blob/release-20200902/activities/workshops/index.md)
[View rendered activities/workshops/mapping-business-models/agenda-template.md](https://github.com/publiccodenet/about/blob/release-20200902/activities/workshops/mapping-business-models/agenda-template.md)
[View rendered activities/workshops/mapping-business-models/index.md](https://github.com/publiccodenet/about/blob/release-20200902/activities/workshops/mapping-business-models/index.md)
[View rendered activities/workshops/mapping-business-models/workshop-guide.md](https://github.com/publiccodenet/about/blob/release-20200902/activities/workshops/mapping-business-models/workshop-guide.md)
[View rendered activities/workshops/mapping-risk-and-service-level-agreements/agenda-template.md](https://github.com/publiccodenet/about/blob/release-20200902/activities/workshops/mapping-risk-and-service-level-agreements/agenda-template.md)
[View rendered activities/workshops/mapping-risk-and-service-level-agreements/index.md](https://github.com/publiccodenet/about/blob/release-20200902/activities/workshops/mapping-risk-and-service-level-agreements/index.md)
[View rendered activities/workshops/mapping-risk-and-service-level-agreements/workshop-guide.md](https://github.com/publiccodenet/about/blob/release-20200902/activities/workshops/mapping-risk-and-service-level-agreements/workshop-guide.md)
[View rendered contributor-guides/for-staff.md](https://github.com/publiccodenet/about/blob/release-20200902/contributor-guides/for-staff.md)

### CODE REVIEW

File: activities/workshops/mapping-risk-and-service-level-agreements/workshop-guide.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+type: Guide
+explains: How to run the 'Mapping risk and service level agreements' workshop
+---
+
+# How to run the 'Mapping risk and service level agreements' workshop
+
+## Introduction
+
+Work through the agenda, making sure the respective speakers have prepared their bits, and there is ample emphasis on openness and vulnerability, as well as giving each participant an equal voice.
```
Comment (ElenaFdR) [inline activities/workshops/mapping-risk-and-service-level-agreements/workshop-guide.md:10]:
```suggestion
Work through the [agenda](/agenda-template.md), making sure the respective speakers have prepared their bits, and there is ample emphasis on openness and vulnerability, as well as giving each participant an equal voice.
```

File: activities/workshops/mapping-risk-and-service-level-agreements/workshop-guide.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Breakout workshops
+
+Explain the canvas and make sure everyone has a chance to ask questions.
```
Comment (ElenaFdR) [inline activities/workshops/mapping-risk-and-service-level-agreements/workshop-guide.md:14]:
```suggestion
Explain the [canvas](/mapping-risk-and-service-level-agreements-template.pdf) and make sure everyone has a chance to ask questions.
```

File: activities/workshops/mapping-risk-and-service-level-agreements/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+## Goal
+
+Have a collaborative working session with new or existing community members to map out, for a given codebase, potential different risks when using the codebase in production, as well as approaches to service level agreements.
+
+## Considerations
+
+Discussions about service level agreements can get very technical quite quickly.
+Try steer the workshop on what the most important risks are according to the participants.
+Verify this by asking participants to prioritize risks by their impact and likelihood.
+Other questions you can ask is what process can be followed to mitigate these risks.
```
Comment (ElenaFdR) [inline activities/workshops/mapping-risk-and-service-level-agreements/index.md:16]:
```suggestion
Ask what process can be followed to mitigate these risks.
```

File: activities/workshops/mapping-risk-and-service-level-agreements/agenda-template.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+type: Resource
+---
+
+# Template Agenda
```
Comment (ElenaFdR) [inline activities/workshops/mapping-risk-and-service-level-agreements/agenda-template.md:5]:
```suggestion
# Template agenda
```

File: activities/workshops/mapping-risk-and-service-level-agreements/agenda-template.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+* Introduction: Explain the canvas (10 min)
+* Round 1: Risks around the codebase in production (30 mins)
+  * Q1.1) If you relied on the codebase in production, what are the biggest risks you see?
+  * Q1.2) If you relied on the codebase in production, what would be the worst case nightmare crisis scenario?
+  * Q1.3) Impact and likelihood of the risk
+
+* Break: (10 mins)
+
+* Round 2: Service liscence agreements around OpenZaak (40 min)
```
Comment (ElenaFdR) [inline activities/workshops/mapping-risk-and-service-level-agreements/agenda-template.md:29]:
```suggestion
* Round 2: Service license agreements around the codebase (40 min)
```

File: activities/workshops/mapping-business-models/agenda-template.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+---
+type: Resource
+---
+
+# Template Agenda
+
+## Introduction
+
+* 13:00 - 13:05 Welcome & Opening
```
Comment (ElenaFdR) [inline activities/workshops/mapping-business-models/agenda-template.md:9]:
```suggestion
* 13:00 - 13:05 Welcome and opening
```

File: activities/workshops/mapping-business-models/agenda-template.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+---
+type: Resource
+---
+
+# Template Agenda
```
Comment (ElenaFdR) [inline activities/workshops/mapping-business-models/agenda-template.md:5]:
```suggestion
# Template agenda
```

File: activities/workshops/mapping-business-models/agenda-template.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+* Break: Tidy notes (10 mins)
+  * Round 2: Creating business opportunities (40 min)
+    * Q 2.1) For each business opportunity, who could be willing to pay for this?
+    * Q 2.2) What payment mechanisms could be used for this? (e.g.: subscription based, fixed open development budget, bug bounty rewards, innovation vouchers)
+    * Q 2.3) Discussion: What do you need to get started?
+
+## Closing
+
+* 15:30 - 15:40 Break
+* 15:40 - 16:00 Closing, reflections & next steps
```
Comment (ElenaFdR) [inline activities/workshops/mapping-business-models/agenda-template.md:33]:
```suggestion
* 15:40 - 16:00 Closing, reflections and next steps
```

File: activities/workshops/mapping-business-models/workshop-guide.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
+---
+type: Guide
+explains: How to run the 'Mapping business models enabled by a codebase' workshop
+---
+
+# How to run the 'Mapping business models enabled by a codebase' workshop
+
+## Introduction
+
+Work through the agenda, making sure the respective speakers have prepared their bits, and there is ample emphasis on openness and vulnerability, as well as giving each participant an equal voice.
```
Comment (ElenaFdR) [inline activities/workshops/mapping-business-models/workshop-guide.md:10]:
```suggestion
Work through the [agenda](agenda-template.md), making sure the respective speakers have prepared their bits, and there is ample emphasis on openness and vulnerability, as well as giving each participant an equal voice.
```

File: activities/workshops/mapping-business-models/workshop-guide.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+
+## Breakout workshops
+
+Explain the canvas and make sure everyone has a chance to ask questions.
```
Comment (ElenaFdR) [inline activities/workshops/mapping-business-models/workshop-guide.md:14]:
```suggestion
Explain the [canvas](Mapping-business-models-canvas-template.pdf) and make sure everyone has a chance to ask questions.
```

File: activities/workshops/mapping-business-models/index.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+
+## Goal
+
+Have a collaborative working session with new or existing community members to map out potential different business models that can be created around a given codebase.
+
+## Considerations
+
+This can be a difficult and sensitive subject.
+For public organizations and market parties involved, this co-creative process can be a new way of working.
+For market parties, this workshop may mean working toghether with and along side (perceived) competitors.
```
Comment (ElenaFdR) [inline activities/workshops/mapping-business-models/index.md:15]:
```suggestion
For market parties, this workshop may mean working together with and alongside (perceived) competitors.
```

File: activities/workshops/mapping-business-models/index.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+For municipalities, this workshop may mean working through a natural level of distrust or distance vis a vis market parties.
```
Comment (ElenaFdR) [inline activities/workshops/mapping-business-models/index.md:16]:
```suggestion
For municipalities, this workshop may mean working through a natural level of distrust or distance vis-à-vis market parties.
```

File: activities/workshops/index.md [hunk 13]
```diff
@@ -9,5 +9,7 @@ Here is a list of workshops developed or used by the Foundation for Public Code:
 * [Governance exercise](../supporting-codebase-governance/exercise/index.md)
 * [Governance game](../supporting-codebase-governance/game/index.md)
 * [Customized governance workshop](../supporting-codebase-governance/customized-governance-workshop/index.md)
+* [Mapping risk and service level agreements](mapping-risk-and-service-level-agreements/index.md)
+* [Mapping business models enabled by codebase](mapping-business-models/index.md)
```
Comment (ElenaFdR) [inline activities/workshops/index.md:13]:
```suggestion
* [Mapping business models enabled by a codebase](mapping-business-models/index.md)
```

**Review by ElenaFdR — COMMENTED** [review]
Made lots of proofreading suggestions.

**Review by nebster — DISMISSED** [review]
looks good.

**Review by nebster — APPROVED** [review]
let's try this again!  looks good

**Pull request merged**